CMakeLists.txt

```diff
@@ -491,7 +491,16 @@ if (NOT $ENV{S2N_LIBCRYPTO} STREQUAL "awslc")
     target_compile_options(${PROJECT_NAME} PRIVATE -Wcast-qual)
 endif()
 
-#work around target differences
+# For interning, we need to find the static libcrypto library. Cmake configs
```

Comment:
We can move this around the find_package call so that it's easier to see which function we're trying to affect.